### PR TITLE
Some improvements to the populate_mets script

### PR DESCRIPTION
**Pull request by alicefuzier**
- [x] Make it able to restart at an arbitrary key in the table
- [x] Make it less verbose
- [x] Make it pause to allow to refresh the token if the credentials expire

### CODE REVIEW

File: mets_adapter/populate_mets/populate_mets.py [hunk 1]
```diff
@@ -60,16 +77,31 @@ def publish(self, space, id):
...
         num_records = maxsize
-    scanner = StorageManifestScanner()
+
+    scanner = StorageManifestScanner(start_record, num_records)
     publisher = MessagePublisher()
-    for i, (space, id) in zip(range(1, num_records + 1), scanner.scan()):
-        click.echo(f"{click.style(str(i), fg='blue')}: {space}/{id}")
-        publisher.publish(space, id)
+    try:
+        for i, (space, id) in zip(range(1, num_records + 1), scanner.scan()):
```
Comment (warrd) [inline mets_adapter/populate_mets/populate_mets.py]: Thought I could have spotted a bug here with it overflowing if `num_records == sys.maxsize`, but apparently not:

```
In [3]: sys.maxsize                                                                                                                                                                                                                                                              
Out[3]: 9223372036854775807

In [4]: sys.maxsize + 1                                                                                                                                                                                                                                                          
Out[4]: 9223372036854775808

In [5]: type(sys.maxsize)                                                                                                                                                                                                                                                        
Out[5]: int

In [6]: type(sys.maxsize + 1)                                                                                                                                                                                                                                                    
Out[6]: int
```

Turns out `sys.maxsize` is not the biggest `int` after all!

Comment (alicefuzier) [reply mets_adapter/populate_mets/populate_mets.py]: 🙄 of course...

Comment (warrd) [reply mets_adapter/populate_mets/populate_mets.py]: https://stackoverflow.com/a/48138759

**Comment by kenoir** [comment]
@alicefuzier why is this still open?

**Comment by alicefuzier** [comment]
I don't think this script is needed anymore, so I'm going to close this PR and open another one to delete it

**Pull request closed without merging**